.github/workflows/ci.yaml

```diff
@@ -35,14 +35,13 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: ['8.0', '8.1']
+                php: ['8.0', '8.1', '8.2', '8.3']
```

Comment:
Thought we already can also add 8.3 to the test range. In the composer.json its already allowed via `^8.0`